pycode/memilio-epidata/memilio/epidata/getDataIntoPandasDataFrame.py

```diff
@@ -77,23 +77,32 @@ def loadGeojson(
 
 def loadCsv(
         targetFileName, apiUrl='https://opendata.arcgis.com/datasets/',
-        extension='.csv', encoding=None):
+        extension='.csv', param_dict={}):
     """! Loads data sets in CSV format. (pandas DataFrame)
     This routine loads data sets (default from ArcGIS) in CSV format of the given public data
     item ID into a pandas DataFrame and returns the DataFrame.
 
     @param targetFileName -- file name which should be downloaded, for ArcGIS it should be public data item ID (string)
     @param apiUrl -- API URL (string, default
               'https://opendata.arcgis.com/datasets/')
-    @param extension -- Data format extension (string, default 'csv')
-    @param encoding -- CSV Encoding format (string, default None)
-    return dataframe
+    @param extension -- Data format extension (string, default '.csv')
+    @param param_dict -- Defines the parameter for read_csv:
+            "sep": Delimiter to use (string, default ',')
+            "header": Row to use for column labels (Use None if there is no header) (int, default 0)
+            "encoding": Encoding to use for UTF when reading (string, default None)
+            "dtype": Data type for data or column (dict of column -> type, default None)
+    @return dataframe 
     """
 
     url = apiUrl + targetFileName + extension
+    param_dict_default = {"sep": ',', "header": 0, "encoding": None, 'dtype': None}
+
+    for k in param_dict_default:
```

Comment:
I like the idea of including " param_dict" to keep it more readable. However, the dict is often initialized inside the constructor, which in my opinion has an opposite effect. 

If keyword args with default values are used directly instead, we would save that part of the code.